src/en/help-openstack.md

```diff
@@ -20,6 +20,10 @@ juju add-cloud
 ```
 ...will enter the interactive mode. Enter the desired values to continue.
 
+!!! Note:
+To use the authentication from OpenStack, find the URL in your novarc the
```

Comment:
Thanks for adding extra info. I read this 3 times though and it still sounds badly worded. Perhaps:

If you have a `novarc` file for your OpenStack, the information you require will be found in there


Though also:
 * it is really jarring to have this note in the middle of the steps
 * If they have a .novarc file, they should just use autoload-credentials